code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1546,6 +1546,18 @@
 	M.update_transform()
 	..()
 
+/datum/reagent/plastic_polymers
+	name = "plastic polymers"
+	id = "plastic_polymers"
+	description = "the petroleum based components of plastic"
+	color = "#f7eded"
+	taste_description = "plastic"
+
+/datum/reagent/plastic_polymers/on_mob_life(mob/living/M)
+	M.adjustToxLoss(1, 0)
+	. = 1
+	..()
+
```

Comment:
Well that's just my working excuse but since you mention it, sure ill just remove it before a maintainer blocks me with a crate of oranges requesting changes.